pyincore/globals.py

```diff
@@ -27,7 +27,9 @@
 PYINCORE_USER_CACHE = os.path.join(USER_HOME, USER_CACHE_DIR)
 
 DATA_CACHE_FOLDER_NAME = "cache_data"
+DATA_CACHE_HASH_NAMES_SERVICE_JSON = "service.json"
 PYINCORE_USER_DATA_CACHE = os.path.join(PYINCORE_USER_CACHE, DATA_CACHE_FOLDER_NAME)
+PYINCORE_SERVICE_JSON = os.path.join(PYINCORE_USER_CACHE, DATA_CACHE_HASH_NAMES_SERVICE_JSON)
 
 TOKEN_FILE_NAME = ".incoretoken"
```

Comment:
Should this be removed?

Reply:
No, actually we need the path to access the file so `PYINCORE_SERVICE_JSON` would be needed. Did you have anything else in mind?